src/components/ContactDetails/ContactDetailsAvatar.vue

```diff
@@ -26,7 +26,7 @@
 		<div class="contact-header-avatar__wrapper">
 			<div class="contact-header-avatar__background" @click="toggleModal" />
 			<div v-if="contact.photo"
-				:style="{ 'backgroundImage': `url(${contact.photoUrl})` }"
+				:style="{ 'backgroundImage': `url(${contact.photoUrl})`, 'backgroundColor': '#fff' }"
```

Comment:
Better in the direct scss file rather than inline if it's not computed

Reply:
Good point – fixed